CONTRIBUTING.md

```diff
@@ -47,44 +48,47 @@ the expected behavior that doesn't occur.
 
 To suggest an enhancement, open an
 [issue](https://github.com/nginx/unit/issues/new) on GitHub with the label
-`enhancement`.  Please do this before implementing a new feature to discuss the
-feature first.
+`enhancement`.  Please do this before implementing a new feature to discuss
+the feature first.
 
 
 ### Open a Pull Request
 
-Before submitting a PR, please read the NGINX Unit code guidelines to know more
-about coding conventions and benchmarks.  Fork the repo, create a branch, and
-submit a PR when your changes are tested and ready for review.  Again, if you'd
-like to implement a new feature, please consider creating a feature request
-issue first to start a discussion about the feature.
+Before submitting a PR, please read the NGINX Unit code guidelines to know
+more about coding conventions and benchmarks.  Fork the repo, create a branch,
+and submit a PR when your changes are tested and ready for review.  Again, if
+you'd like to implement a new feature, please consider creating a feature
+request issue first to start a discussion about the feature.
 
 
 ## Git Style Guide
 
-- Keep a clean, concise and meaningful `git commit` history on your branch,
-  rebasing locally and squashing before submitting a PR
+- Create atomic commits.  A commit should do just one thing, I.e you
```

Comment:
Double space, and i.e. is lowercase when in the middle of the sentence, like here, and because it's an abbreviation, the periods are needed after both i and e. See https://www.grammarly.com/blog/know-your-latin-i-e-vs-e-g/
```suggestion
- Create atomic commits. A commit should do just one thing, i.e. you
```

Reply:
Is there a reason for that practice in this file? And I assume other files. I understand that double spaces were important when typewriters were around and I'm missing context on whether that or some similar reason made its way to here.

I also want to gently push back on any explanation that sounds like "we've always done it this way." In those cases I'd like to either figure out what the original reason was, see if that's still relevant, and potentially change it if it no longer serves us. 🙂

Reply:
I've no idea, probably personal writing style.